elide-dbmanager/elide-dbmanager-hibernate5/pom.xml

```diff
@@ -0,0 +1,73 @@
+<!--
+  ~ Copyright 2015, Yahoo Inc.
+  ~ Licensed under the Apache License, Version 2.0
+  ~ See LICENSE file in project root for terms.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>elide-dbmanager-hibernate5</artifactId>
+    <packaging>jar</packaging>
+    <name>Elide Hibernate5 Database Manager</name>
+    <description>Elide database manager for Hibernate5 support</description>
+    <parent>
+        <groupId>com.yahoo.elide</groupId>
+        <artifactId>elide-dbmanager-parent-pom</artifactId>
+        <version>1.0.0.1-SNAPSHOT</version>
+    </parent>
+
```

Comment:
Will need to add some fields here for maven central compliance. See: https://github.com/yahoo/elide/pull/13

You can probably just copy all of the blocks directly. Namely, you need the following:

- [ ] JavaDoc and Sources (just include plugins in section)
- [X] Signed artifacts (this is done for us by bintray)
- [ ] Add top-level URL
- [ ] Add license block
- [ ] Add developer block
- [ ] Add scm block